package.json

```
@@ -286,6 +287,7 @@
     "tether-drop": "https://github.com/torkelo/drop/tarball/master",
     "tinycolor2": "1.4.1",
     "tti-polyfill": "0.2.2",
+    "uuid": "^8.3.0",
```

Please use specific versions
```suggestion
    "uuid": "8.3.0",
```